crates/modelardb_server/src/storage/mod.rs

```diff
@@ -344,6 +347,30 @@ impl StorageEngine {
         ]
     }
 
+    /// Update the remote data folder used to transfer data in the data transfer component.
```

Comment:
Seem something is missing from this sentence, maybe `... used to transfer data to in the...`.

Reply:
Fixed.